docs/features/search/search-engines.md

```diff
@@ -239,6 +239,26 @@ search:
 
 > You can also increase the batch size if you are using a large ES instance.
 
+### Elasticsearch batch key field
+
+By default, during bulk uploads with the Elasticsearch indexer, each document is assigned an autogenerated `_id` unless a `batchKeyField` is explicitly set. This configuration is optional and most users won’t need to customize it. However, if your use case involves frequent lookups or updates to existing documents, setting `batchKeyField` can be beneficial. It allows you to define a consistent identifier for each document, helping to streamline updates and prevent duplicate entries. Be aware that if the value provided for `batchKeyField` is not unique across documents, Elasticsearch will overwrite any existing document with the same `_id`.
```

Comment:
You can add the words that vale isn't happy with to the list here to resolve this errors: https://github.com/backstage/backstage/blob/master/.github/vale/config/vocabularies/Backstage/accept.txt